mol/molecule.py

```diff
@@ -3354,6 +3374,8 @@ def add_hatoms(self) -> "Molecule":
                 )
                 p.communicate()
                 retmol = self.__class__(f_out.name)
+                if any(self.bonds):
+                    retmol.guess_bonds()
```

Comment:
I do not understand why we should guess bonds by default. I would remove this line.

Reply:
Previously the behaviour was that if you do `add_hatoms` to a molecule that already has bonds, the result will have no bonds at all, as it recreates a molecule from an xyz. This change means that if the previous molecule has bonds, it will guess them. But I can remove it if you think it is unexpected here.

Reply:
ah true it makes sense.

Reply:
I have found a way to get the bonding information out of `amsprep`, so have used this instead of guess bonds

(note the lint failure is due to new mypy issues, being fixed on another branch)